tests/php/assert_pkg_ca_core_3_3.php

```
@@ -0,0 +1,218 @@
+{
+	return $page === 'pkg_mgr_installed.php' && $GLOBALS['allowed'];
+}
+
+function is_subsystem_dirty(string $name): bool
+{
+	return $name === 'pkg' && $GLOBALS['dirty'];
+}
+
+function pkg_version_compare(string $left, string $right): string
+{
+	return version_compare($left, $right);
```

PHPStan (2.2.7)

[error] Function pkg_version_compare() should return string but returns int.


(return.type)